prometheus/base/prometheus.rules.yaml

```
@@ -1,13 +1,12 @@
-apiVersion: monitoring.coreos.com/v1
-kind: PrometheusRule
-metadata:
-  labels:
-    openshift.io/prometheus-rule-evaluation-scope: leaf-prometheus
-  name: o11y-example-rule
-  namespace: o11y
-spec:
-  groups:
-    - name: example
-      rules:
-        - record: o11y_prometheus_http_requests_total
-          expr: sum by (code) (prometheus_http_requests_total)
+
+groups:
+  - name: stonesoup-metering
+    interval: 30s
+    rules:
+       - record: container:network_transmit_bytes:group
+         expr: |2
```

```suggestion
         expr: |
```
As far as I understand, `|2` means that you should have an indentation of `2` for content of `expr`. In reality, I think you have more. I think you can just drop the number for now.

See some details:
https://stackoverflow.com/questions/15540635/what-is-the-use-of-the-pipe-symbol-in-yaml

---

Done :)